tests/DependencyLoaderTest.php

```diff
@@ -6,28 +6,55 @@
 
 use Dominikb\ComposerLicenseChecker\Contracts\DependencyParser;
 use Dominikb\ComposerLicenseChecker\DependencyLoader;
+use Dominikb\ComposerLicenseChecker\Exceptions\CommandExecutionException;
 use Mockery;
+use Symfony\Component\Console\Command\Command;
 
 class DependencyLoaderTest extends TestCase
 {
-    /** @test */
+    /**
+     * @test
+     *
+     * @requires OS Linux
+     * Linux required because of escape characters in the verified command
+     */
     public function it_runs_the_command_with_the_given_inputs()
     {
         $loader = Mockery::mock(DependencyLoader::class, [$this->createNoOpParser()])
                          ->makePartial();
 
         $command = '';
         $loader->shouldAllowMockingProtectedMethods()
-               ->shouldReceive('exec')
-               ->once()
+               ->expects('exec')
                ->withArgs(function ($c) use (&$command) {
                    return (bool) ($command = $c);
                })
-               ->andReturn([]);
+               ->andReturns([]);
 
         $loader->loadDependencies('./composerpath/composer-binary', '/some/directory');
 
-        $this->assertEquals('./composerpath/composer-binary licenses -f json -d /some/directory', $command);
+        $this->assertEquals('\'./composerpath/composer-binary\' licenses --format json --working-dir \'/some/directory\'', $command);
```

Comment:
Let's make use of double quotes to avoid the escaping in the assertion.

```suggestion
        $this->assertEquals("'./composerpath/composer-binary' licenses --format json --working-dir '/some/directory'", $command);
```